jaxrs/WFLY-11943_EAP7-606_Statistics_of_endpoints.adoc

```diff
@@ -0,0 +1,78 @@
+= EAP7-606 / Statistics of REST endpoints
+:author:            Rebecca Searls
+:email:             rsearls@redhat.com
+:toc:               left
+:icons:             font
+:idprefix:
+:idseparator:       -
+:issue-base-url:    https://issues.jboss.org/browse
+
+== Overview
+
+Collect and report runtime statistics on REST endpoints running in RESTEasy.
+
+== Issue Metadata
+
+=== Issue
+
+* {issue-base-url}/WFLY-11943[WFLY-11943]
+* {issue-base-url}/EAP7-606[EAP7-606]
+
+=== Related Issues
+
+* https://issues.jboss.org/browse/WFLY-11751[WFLY-11751]
+
+=== Dev Contacts
+
+* mailto:asoldano@redhat.com[Alessio Soldano]
+* mailto:ema@redhat.com[Jim Ma]
+* mailto:rsigal@redhat.com[Ron Sigal]
+* mailto:rsearls@redhat.com[Rebecca Searls]
+* mailto:weli@redhat.com[Wei Nan Li]
```

Comment:
@rsearls , you should be the main/first dev contact in the list here, followed by Ron and me.